joss/neostjoss.bib

```diff
@@ -384,6 +384,16 @@ @misc{Scipy
     note = {[Online; accessed 21.06.2019]}
 }
 
+@misc{aliveprogress,
+    author = {Sampaio de Almeida, Rog\'{e}rio},
+    title - {alive-progress},
+    year = {2019 --},
```

Comment:
Just noticed I used title - instead of title =, can you correct this in review @thjsal ?!

Reply:
Done!